lib/video_info/providers/vkontakte.rb

```diff
@@ -75,19 +90,19 @@ def _api_path
       end
 
       def _data_hash
-        data[/hash2\\":\\"(\w+)/, 1]
+        data[/"hash2":"(\w+)/, 1]
       end
 
       def _set_video_id_from_url
-        url.gsub(_url_regex) { @video_owner, @video_id = $1.split('_') }
+        url.gsub(_url_regex) { @video_owner, @video_id = $1.split("_") }
```

Comment:
Prefer single-quoted strings when you don't need string interpolation or special symbols.